File: 1344_angles_between_a_clock/angle_clock.go
package main

import "math"

// Time: O(1)
// Space: O(1)
func angleClock(hour int, minutes int) float64 {
	var h, m float64
	h = float64(30*hour) + float64(minutes/2)
	m = 6 * float64(minutes)
	diff := math.Abs(h - m)
	return math.Min(diff, 360-diff)
}

func main() {

}

File: 1344_angles_between_a_clock/angle_clock.py
def angle_clock(hour: int, minutes: int) -> float:
    """
    Time: O(1)
    Space: O(1)
    """
    h = ((360/12) * hour) + (360/12/60) * minutes
    m = (360/60) * minutes
    diff = abs(h - m)
    return diff if diff <= 180 else 360 - diff
